pushapkscript/googleplay.py

```diff
@@ -1,10 +1,13 @@
+from pushapkscript.exceptions import TaskVerificationError
 from pushapkscript.task import extract_channel
 
 # TODO Change the "aurora" scope to "nightly" so we can use the dict defined in mozapkpublisher
 CHANNEL_TO_PACKAGE_NAME = {
     'aurora': 'org.mozilla.fennec_aurora',
     'beta': 'org.mozilla.firefox_beta',
     'release': 'org.mozilla.firefox',
+    # dep-signing mimics Aurora
+    'dep': 'org.mozilla.fennec_aurora',
```

Comment:
You asked about this in email -- is there a way to allow for * in dep? We're going to want to support people pushing to try with dep signing, and by default they might get org.mozilla.fennec_USER iirc. We can fix this here, or in a followup.